qemu/tests/cfg/win_virtio_driver_install_from_update.cfg

```diff
@@ -126,3 +126,8 @@
             device_name = "VirtIO FS Device"
             device_hwid = '"PCI\VEN_1AF4&DEV_105A"'
             fs_binary_extra_options = " -o cache=auto"
+        - with_fwcfg:
+            vmcoreinfo = yes
```

Comment:
@leidwang Could you limit the guests and hosts that fwcfg support?

Reply:
There is a limitation on the top, it's okay with me.   
 only Windows
    no WinXP WinVista Win7 Win8 Win8.1 Win2003
    no Win2008 Win2008..r2 Win2012 Win2012..r2